### DISCUSSION TITLE
Missing data on Drive Stats

### DISCUSSION THREAD

**Discussion by llgiordano** [teslamate-org/teslamate#2034] [Q&A]
![image](https://user-images.githubusercontent.com/91390170/136009373-30db59f0-dc58-4fa0-8010-72e84d921095.jpeg)

Since upgrading to v1.24.2, I have several "no data" in the dashboard Drive Stats.

The problem is on both my cars

The problem happens independently on the selected time frame

Installation is on Docker / RaspberryPi 

All the rest works properly 

Any idea about what I'm doing wrong?

**Comment by curlybraces** [comment]
Same thing happened to me and I was about to submit a bug report. Maybe downgrading back to version 1.24.1 will fix it?

**Reply by Otsirc** [reply]
Exact same issue here on "drive stats", since latest update.
Additionally i also notice that on "updates" it does not show charges or rated range for latest update, although there are plenty logged in reality.
![image](https://user-images.githubusercontent.com/46891766/136336509-7d678d87-2796-499d-9f9c-673d8c74ea7a.png)

**Reply by llgiordano** [reply]
Yes, same issue on the updates page also for me![image](https://user-images.githubusercontent.com/91390170/136375391-98b34dbf-0a3b-4d2a-ba77-2968abadc14a.jpeg)

**Comment by curlybraces** [comment]
I updated my OS to the newest version as well as docker and docker-compose. Everything is working after I did all of the updates and rebooting. Good luck!

**Reply by Otsirc** [reply]
Strange, for me it did not work.
I backed up the DB, removed Docker, containers, images etc...
update OS, reinstalled Teslamate, imported DB, and same result.

**Reply by llgiordano** [reply]
Also still not working for me, update did not solved the problem

**Reply by natrimed** [reply]
Also having missing data. The drives are an example; consumption missing. As a result it does not populate the cost on the other dashboard.

Latest version being used in Docker on a Synology NAS.

<img width="1440" alt="Screen Shot 2021-10-13 at 5 09 43 PM" src="https://user-images.githubusercontent.com/92419211/137067074-5363278f-0965-4b52-9a83-e512c245aea2.png">

**Reply by llgiordano** [reply]
I have updated teslamate once more and the problem is now solved. The issue can be closed for me, thanks @adriankumpf
